homeassistant/components/fastdotcom/sensor.py

```diff
@@ -0,0 +1,71 @@
+"""
+Support for Fast.com internet speed testing sensor.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/sensor.fastdotcom/
+"""
+import logging
+
+from homeassistant.components.fastdotcom import DOMAIN as FASTDOTCOM_DOMAIN, \
+    DATA_UPDATED
+from homeassistant.helpers.dispatcher import async_dispatcher_connect
+from homeassistant.helpers.restore_state import RestoreEntity
+
+DEPENDENCIES = ['fastdotcom']
+
+_LOGGER = logging.getLogger(__name__)
+
+ICON = 'mdi:speedometer'
+
+
+def setup_platform(hass, config, add_entities, discovery_info=None):
```

Comment:
We can make everything about this platform async, since there's no I/O being done anywhere.